seacatauth/batman/elasticsearch.py

```
@@ -398,19 +423,23 @@ async def sync_credentials(self, session: aiohttp.ClientSession, cred: dict):
+				if 200 <= resp.status < 300:
+					# Everything is alright here
+					pass
+				else:
+					text = await resp.text()
+					L.warning(
+						"Failed to create/update user in ElasticSearch:\n{}".format(text[:1000]),
+						struct_data={"cid": cred["_id"]}
+					)
+		except aiohttp.client_exceptions.ClientConnectionError:
+			raise aiohttp.client_exceptions.ClientConnectionError(
+				"Cannot connect to any of the configured ElasticSearch nodes.")
```

Ruff (0.14.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)